### PR TITLE
Change gen-class-name for fonts colors assets [PR-280](https://github.com/FlutterGen/flutter_gen/pull/280)

### PR DESCRIPTION
**Pull request by Cu-Toof**

### CODE REVIEW

**Comment by Cu-Toof** [comment]
@thangnc Please review it

**Comment by wasabeef** [comment]
@Cu-Toof 
Thank you for your PR. plase fix the conflicts.

File: packages/core/test_resources/pubspec_assets_exclude_files.yaml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    - assets/images/
+    - assets/images/chip3/chip3.jpg
+    - assets/images/chip3/chip3.jpg # duplicated
+    - assets/images/chip4/
+    - assets/images/icons/fuchsia.svg
+    - assets/images/icons/kmm.svg
+    - assets/images/icons/paint.svg
+    - assets/images/icons/dart@test.svg
+    - assets/json/fruits.json
+    - pictures/chip5.jpg
```
Comment (thangnc) [inline packages/core/test_resources/pubspec_assets_exclude_files.yaml:28]: Please add 1 new line for EOF here

**Review by thangnc — COMMENTED** [review]
Basically LGTM except should rebase on `main` branch. I left 1 minor comment

File: packages/core/test_resources/pubspec_assets_exclude_files.yaml (hunk 1 above)
Comment (Cu-Toof) [reply packages/core/test_resources/pubspec_assets_exclude_files.yaml]: @thangnc I have updated PR.

Comment (Cu-Toof) [reply packages/core/test_resources/pubspec_assets_exclude_files.yaml]: @thangnc Please review my PR 
If everything is ok, please notify me and @wasabeef Could you merge my PR?

**Review by thangnc — APPROVED** [review]
LGTM. @wasabeef can you help to merge this PR?

**Comment by wasabeef** [comment]
@Cu-Toof @thangnc
Thank you! I just merged this PR.

@thangnc 
If you can continue the code review for FlutterGen elsewhere PRs, I’m looking forward to continuing working with you.

**Comment by thangnc** [comment]
@wasabeef thks for trusting me. I will take a look on other PRs

**Pull request merged**